toolchains/nightly/rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
 channel = "nightly-2024-07-11"
-components = [ "clippy", "rustfmt" ]
+components = [ "clippy", "rustfmt", "rust-src" ]
```

Comment:
Is that required by `wasm_threads`?

Reply:
More generally, it's required to be able to recompile the standard library with the additional target features we use for threading. 

I think we already use it elsewhere as well (when building examples maybe?) since the stable toolchain had it already.